src/outfit.ts

```diff
@@ -205,6 +213,28 @@ export function meatOutfit(embezzlerUp: boolean, requirement?: Requirement, sea?
       forceEquip.push($item`Kramco Sausage-o-Matic™`);
     }
   }
+
+  const stickerSlots = $slots`sticker1, sticker2, sticker3`;
+  const UPC = $item`scratch 'n' sniff UPC sticker`;
+
+  if (embezzlerUp) {
+    const currentWeapon = 25 * findLeprechaunMultiplier(meatFamiliar());
+    const embezzlers = globalOptions.ascending
+      ? 20
+      : Math.min(30, embezzlerCount() || digitizedMonstersRemaining());
+
+    const addedValueOfFullSword = (embezzlers * ((75 - currentWeapon) * (750 + baseMeat))) / 100;
+    if (addedValueOfFullSword > mallPrice(UPC)) {
```

Comment:
If we need to add 3 stickers to get the full value of a full sword, shouldn't this be `3 * mallPrice(UPC)?`